Print git status to debug version tag

AlexanderYastrebov opened the pull request:
Version tag is generated based on the `git describe --tags --always --dirty`.
Starting from `9a6ffb4-dirty` it has `-dirty` suffix signalling that
working tree has local modification.

Signed-off-by: Alexander Yastrebov <alexander.yastrebov@zalando.de>

szuecs commented on delivery.yaml:
```diff
@@ -20,6 +20,7 @@ pipeline:
       make build.docker
   - desc: push
     cmd: |
+      git status
```
why not `git diff` if VERSION has "dirty"?
```
echo $VERSION | grep -q dirty && git diff
```

AlexanderYastrebov replied:
git diff shows only upstaged changes and I was not sure what was changed. git status gives a better overview https://github.com/zalando-incubator/cluster-lifecycle-manager/pull/544#issuecomment-998050897

AlexanderYastrebov commented:
```
+ git status
HEAD detached from a41f4ef
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   pkg/cluster-registry/client/clusters/list_clusters_responses.go
	modified:   pkg/cluster-registry/client/infrastructure_accounts/list_infrastructure_accounts_responses.go
	modified:   pkg/cluster-registry/client/node_pools/list_node_pools_responses.go
	modified:   pkg/cluster-registry/models/cluster.go
	modified:   pkg/cluster-registry/models/cluster_status.go
	modified:   pkg/cluster-registry/models/cluster_update.go
no changes added to commit (use "git add" and/or "git commit -a")
```

AlexanderYastrebov commented:
It looks like we need to regenerate cluster registry api
